ouroboros-consensus-cardano-tools/app/DBAnalyser/Parsers.hs

```diff
@@ -1,8 +1,12 @@
 {-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE CPP           #-}
 
 module DBAnalyser.Parsers (parseCmdLine) where
 
+#if __GLASGOW_HASKELL__ < 900
```

Comment:
Could we avoid CPP here if we used a qualified import?

A very rushed experiment suggests that making this a qualified import and similarly qualifying the `asum` occurrences in this module (I assume there aren't many) would avoid the warning.